api/src/main/java/com/codeforcommunity/dto/report/ReportWithoutLibrary.java

```diff
@@ -19,23 +24,40 @@ public ReportWithoutLibrary(
       Boolean hasSpace,
       String currentStatus,
       String reason,
-      String timeline) {
-    super(id, schoolId, userId, numberOfChildren, numberOfBooks, mostRecentShipmentYear);
+      ReadyTimeline timeline) {
+    super(
+        id,
+        createdAt,
+        updatedAt,
+        schoolId,
+        userId,
+        numberOfChildren,
+        numberOfBooks,
+        mostRecentShipmentYear);
     this.wantsLibrary = wantsLibrary;
     this.hasSpace = hasSpace;
     this.currentStatus = currentStatus;
     this.reason = reason;
     this.timeline = timeline;
   }
 
-  public boolean getWantsLibrary() { return this.wantsLibrary; }
-
-  public boolean getHasSpace() { return this.hasSpace; }
+  public boolean getWantsLibrary() {
```

Comment:
for consistency should this be returning a type Boolean instead of boolean?

Reply:
Yes sir good catch